doc/source/user_guide/new_meshing_workflows.rst

```diff
@@ -0,0 +1,462 @@
+.. _ref_user_guide_new_meshing_workflows:
+
+Use meshing workflows
+=====================
+PyFluent supports accessing all Fluent meshing functionalities, including
+guided meshing workflows.
+
+Watertight geometry meshing workflow
+------------------------------------
+This simple example shows how to use the watertight geometry meshing workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('mixing_elbow.pmdb', 'pyfluent/mixing_elbow')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    watertight = meshing.watertight()
+    watertight.import_geometry.FileName.set_state(import_file_name)
+    watertight.import_geometry.LengthUnit.set_state('in')
+    watertight.import_geometry.Execute()
+
+Add local sizing
+~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.add_local_sizing.AddChildToTask()
```

Comment:
We don't want to expose anything like this, but there is something to be aware of, as follows. We have code like the following in the meshing workflow tests. But this relies on AddChild which is a specific argument in meshing datamodel.
```
    add_local_sizing = watertight.add_local_sizing
    add_local_sizing.add_child(state={"BOIFaceLabelList": ["cold-inlet"]})
```
This is the code in workflow.py. As I say AddChild is specific to meshing.py:
```
    def add_child(self, state: Optional[dict] = None) -> None:
        """Add a child to this CompoundTask.

        Parameters
        ----------
        state : Optional[dict]
            Optional state.
        """
        state = state or {}
        state.update({"AddChild": "yes"})
        self._task.Arguments.set_state(state) 
```
and then
```
    def add_child_and_update(self, state=None):
        """Add a child to this CompoundTask and update.

        Parameters
        ----------
        state : Optional[dict]
            Optional state.
        """
        self.add_child(state)
        self._task.AddChildAndUpdate()
        return self.last_child()
```
Ignoring for a moment the coupling with the meshing data model, you should look into hiding `AddChildToTask` just as for `AddChildAndUpdate`. `AddChildToTask` is simpler.